st2common/st2common/router.py

```diff
@@ -669,7 +669,7 @@ def process_item(item):
                 result.append(item)
         elif isinstance(data, dict):
             # get_one response
-            result = process_item(item)
+            result = process_item(data)
```

Comment:
Good catch. It looks we don't have test case for this scenario? :)

Reply:
Besides that and test case for handling of those attributes it looks good to me :+1: 

I believe we already have a pattern for include and exclude attributes test cases. We might not just have one for "get one" API endpoints yet.